lib/roo_helper/loader.rb

```diff
@@ -171,9 +171,11 @@ def merge_options(file_options)
 
     # Set CSV options
     # See https://apidock.com/ruby/v2_5_5/CSV/new/class for more options
+    # Forces encoding to be utf-8
     def csv_options(file, file_options)
       csv_options = {
-        col_sep: csv_col_sep(file, file_options)
+        col_sep: csv_col_sep(file, file_options),
+        encoding: 'ISO-8859-1',
```

Comment:
I'm confused. Shouldn't this be `utf-8` here instead of `ISO-8859-1`?

I see [the docs recommending](https://github.com/roo-rb/roo#csv-support) `bom|utf-8`.

Reply:
While the previous gem used [utf-8] (https://github.com/tilo/smarter_csv/blob/1.2-stable/lib/smarter_csv/smarter_csv.rb#L47) this did not work for new gem, neither did `bom|utf-8` as the encoding

`ISO-8859-1` does not affect clean csv files made in a text editor

`csv_options` are only passed to `Roo::Spreadsheet.open` if it is a `.csv` or `.txt` file
https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/lib/roo_helper/loader.rb#L33